src/main/webapp/components/ControlWrapper.jsx

```diff
@@ -33,14 +33,22 @@ class ControlWrapper extends React.PureComponent {
         this.controlType = this.isString(props.entity.type)
             ? CONTROL_TYPE_MAP[props.entity.type]
             : null;
+        // Add 'optional' placeholder for optional field
+        this.options = this.props.entity.options;
+        if (this.props.entity.required === false || this.props.entity.required === undefined) {
```

Comment:
Can we get away by just checking `if(!this.props.entity.required)`?

Reply:
Done